### DISCUSSION TITLE
Recognize xx.setup as url

### DISCUSSION THREAD

**Discussion by EggToaster** [Floorp-Projects/Floorp#875] [Feature Request]
currently, you have to type https://xx.setup to access but if you can access just using xx.setup, it will be helpful.

**Comment by regularentropy** [comment]
@TV-Toshikawa Could you please add more details on this, since it _technically_ is a url

**Reply by EggToaster** [reply]
If i type buffalo.setup, it will be search instead of opening as url.

**Reply by regularentropy** [reply]
Expected behavior, since there is no TLD ((top-level domain), named .setup
